deepmedic/logging/tensorboard_logger.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import, print_function, division
+
+import tensorflow as tf
+
+
+class TensorboardLogger(object):
+
+    def __init__(self, log_path, tf_graph):
+        self.logger = tf.compat.v1.summary.FileWriter(log_path, tf_graph)
```

Comment:
This line threw error with Tensorflow 1.12.0 (no such module).

Why not:
```suggestion
        self.logger = tf.summary.FileWriter(log_path, tf_graph)
```
??

This one works fine for me. Not sure if something changes in later TF versions.